<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-3QCSFSN" failures="1" tests="5" name="Test" time="3.432" errors="0" timestamp="2024-03-18T19:38:31 IST">
  <testcase classname="testcases.Getrecordbasedonthecustomerid" name="getrecordbasedontheid" time="2.566">
    <failure type="java.lang.AssertionError" message="expected [200] but found [404]">
      <![CDATA[java.lang.AssertionError: expected [200] but found [404]
at testcases.Getrecordbasedonthecustomerid.getrecordbasedontheid(Getrecordbasedonthecustomerid.java:32)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 34 stack frames]]>
    </failure>
  </testcase> <!-- getrecordbasedontheid -->
  <testcase classname="testcases.Addcustomer" name="addcustomer" time="0.102"/>
  <testcase classname="testcases.GetAllTheCustomers" name="getallthecustomers" time="0.045"/>
  <testcase classname="testcases.deletecustomer" name="deletethecustomers" time="0.051"/>
  <testcase classname="testcases.GetAllTheCustomers_requestes" name="getallthecustomers" time="0.521"/>
</testsuite> <!-- Test -->
